python/cudf/cudf/benchmarks/cuio_release_deck_bench.py

```diff
@@ -0,0 +1,171 @@
+import cudf

```

Comment:
Needs copyright.

Can we add a docstring at the top of this module describing what is being benchmarked here? This is to prevent other, unrelated benchmarks from ending up here.